pulsar-client/src/main/java/org/apache/pulsar/client/impl/auth/oauth2/Readme.md

```diff
@@ -0,0 +1,94 @@
+# Pulsar Client Authentication Plugin for OAuth 2.0
```

Comment:
@jiazhai Can you move this README to the website documentation?

Reply:
thanks @sijie, already marked it as doc-required. will provide another Pr to refine the doc.